src/features/widgetViews/ui/Radar/widget/views/default/index.tsx

```diff
@@ -0,0 +1,82 @@
+import React, {useRef} from 'react';
+import {
+    Legend,
+    PolarAngleAxis,
+    PolarGrid,
+    PolarRadiusAxis,
+    Radar,
+    RadarChart,
+    ResponsiveContainer,
+    Tooltip
+} from 'recharts';
+
+import {Copyrights} from '@/shared/ui/Copyrights';
+import {Module} from '@/shared/ui';
+import {getColorsFromPaletteByVariant, PaletteVariant} from '@/shared/ui/styles/palettes';
+import {useChildRatio} from '@/shared/ui/hooks/useChildRatio';
+import Loader from '@/shared/ui/Loader';
+import {CopyrightsRaw} from '@/shared/api/dotsight';
+
+import {Tooltip as CustomizedTooltip} from '../../components/Tooltip';
+import {Legend as CustomizedLegend} from '../../components/Legend';
+import {Tick as CustomTick} from '../../components/Tick';
+import {BaseProps} from '..';
+import styles from './index.module.scss';
+
+export const configuration = {
+    width: 2,
+    height: 1,
+};
+
+type Props = BaseProps & {
+    copyrights: CopyrightsRaw,
+}
+
+export const View = ({items, chart, copyrights, metrics}: Props) => {
+    const rootRef = useRef(null);
+    const legendContainerRef = useRef(null);
+    const copyrightsContainerRef = useRef(null);
+    const {ratio, isInitiated} = useChildRatio({
+        parentRef: rootRef,
+        extraChildrenRefs: [legendContainerRef, copyrightsContainerRef],
+        parentRatio: configuration.width / configuration.height,
+    });
+    return (
+        <Module className={styles.root} ref={rootRef}>
+            <ResponsiveContainer width="100%" aspect={ratio}>
```

Comment:
looks like this is going to be the same for every View, we should deduplicate the code somehow

Reply:
But next view can possibly be constructed differently. For example with another chart library. Right now I see no reason to make stronger abstractions